permissions/test-results-aggregator.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "test-results-aggregator"
+github: "jenkinsci/test-results-aggregator-plugin"
+paths:
+- "com/jenkins/testresultsaggregator"
```

Comment:
The path here is incorrect. It should be the groupId / artifactId from your pom.xml (https://github.com/jenkinsci/test-results-aggregator-plugin/blob/master/pom.xml#L11 for groupId and https://github.com/jenkinsci/test-results-aggregator-plugin/blob/master/pom.xml#L12 for artifactId).

In your case the groupId is io.jenkins.plugins and the artifactId is test-results-aggregator, so the path would be:

`io/jenkins/plugins/tests-results-aggregator`

Reply:
Thanks , i just change paths io/jenkins/plugins/tests-results-aggregator